gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/internal/KeelService.groovy

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.services.internal
+
+import retrofit.http.Body
+import retrofit.http.DELETE
+import retrofit.http.GET
+import retrofit.http.Headers
+import retrofit.http.POST
+import retrofit.http.Path
+import retrofit.http.Query
+
+interface KeelService {
+
+  @GET("/v1/intents/{id}")
+  Map getIntent(@Path("id") String intentId)
+
+  @GET("/v1/intents")
+  List<Map> getIntents(@Query("status") List<String> status)
```

Comment:
@robzienert `keel` is still a baby, any thoughts on considering envelopes for all responses to better allow for paging, future proofing, etc.?

It's perhaps more relevant to something like clouddriver, but those APIs have more or less sailed for the time being.

A question more than anything.

Reply:
Keel itself has request wrappers, but not response wrappers - probably something worthwhile doing. To your point, however, it probably would make sense to have an external-only wrapper so we can evolve the internal and external APIs independently from each other. 👍 💯

Reply:
I'll add external wrappers.